test/config_SUITE.erl

```diff
@@ -102,6 +102,8 @@
 		    maps:put(handler, 'ergw_aaa_ro', ?RF_CONFIG)},
 	  apps =>
 	      #{<<"RADIUS-Application">> =>
+		    #{'Origin-Host' => <<"dummy.host">>,
+		      procedures =>
 		    #{init => [#{service => <<"Default">>}],
 		      authenticate =>
 			  [maps:merge(?RADIUS_AUTH_CONFIG, ?RADIUS_SERVICE_OPTS),
```

Comment:
the content of `procedures` needs to be reindent.

Reply:
ok